src/components/localization/localized-link.js

```diff
@@ -157,10 +157,10 @@ export const LocalizedLink = React.forwardRef(({ to, ...props }, ref) => {
             </ExternalLink>
         )
 
-    // internal links should end with / e.g. /about/
-    let internal_to = path_to.charAt(to.length - 1) === '/' ? path_to : path_to
+    let internal_to = path_to
 
-    if (props.has_no_end_slash) {
+    // remove trailing character only if it is a forward slash
+    if (props.has_no_end_slash && internal_to.charAt(internal_to.length - 1) == '/') {
         internal_to = internal_to.substring(0, internal_to.length - 1)
     }
```

Comment:
```suggestion
    } else {
      let internal_to = path_to
    }
```

Reply:
Just a suggestion, but would it be more concise to declare it like this?

Reply:
won't we have to define `internal_to` above as let is block scoped?

Reply:
Oh good point, we could instead use a ternary declaration then. Otherwise, it's fine just as it is.

Reply:
tried ternary, looks a little congested. more expressive this way.